isos/bootstrap/bootstrap

```diff
@@ -51,11 +51,11 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     #	rm -f /mnt/.tether/volumes/containerfs
 
     # set up the hostname
-    #	if [ -f /mnt/etc/hostname ]; then
-    #		/bin/hostname -F /mnt/etc/hostname
-    #	else
-    #		/bin/hostname -F /mnt/.tether-init/hostname
-    #	fi
+    if [ -f ${MOUNTPOINT}/etc/hostname ]; then
```

Comment:
` 
# if we don't have a populated init layer, pull from guestinfo
28     if [ ! -f ${MOUNTPOINT}/.tether-init/docker-id ]; then
...
 48     fi
`